algorithm/cgal/getCgal.js

```diff
@@ -33,9 +32,10 @@ export const initCgal = async () => {
         if (path === 'cgal_node.wasm' || path === 'cgal_browser.wasm') {
           let url = import.meta.url;
           if (url.startsWith('file://')) {
-            url = fileURLToPath(url);
-            url = url.substring(0, url.length - 10) + 'cgal_node.wasm';
-            return url;
+            const { pathname } = new URL(url);
```

Comment:
I think this is resulting in the issue with the leading '\' again. I'm seeing "/C:/Users/Bar/Documents/GitHub/JSxCAD4/algorithm/cgal/cgal_node.wasm" as the generated path